lms/djangoapps/onboarding_survey/views.py

```diff
@@ -65,21 +68,25 @@ def user_info(request):
     is then saved. After saving the form, user is redirected to the
     next survey namely, interests survey.
     """
-
+    are_forms_complete = request.user.extended_profile.are_surveys_complete
```

Comment:
Code here needs to be refactored as suggested in other PR #9

Reply:
Will be changed in the next couple of days as discussed.